content/kubermatic/main/tutorials-howtos/operating-system-manager/usage/_index.en.md

```diff
@@ -43,14 +43,14 @@ spec:
 
 To use custom OperatingSystemProfiles, users can do the following:
 
-1. Create their `CustomOperatingSystemProfile` resource in the seed namespace(kubermatic). These resources will be automatically synced to the `kube-system` namespace of the user-clusters.
+1. Create their `CustomOperatingSystemProfile` resource on the seed cluster in the kubermatic namespace. These resources will be automatically synced to the `kube-system` namespace of the user-clusters.
```

Comment:
Nit: Since kube-system is backticked later on, let's do this here as well:

```suggestion
1. Create their `CustomOperatingSystemProfile` resource on the seed cluster in the `kubermatic` namespace. These resources will be automatically synced to the `kube-system` namespace of the user-clusters.
```

Reply:
This should be applied to the other versions as well.

Reply:
nice catch 👍